llvm/include/llvm/Transforms/Yk/NoMoreEntryblockCalls.h

```diff
@@ -0,0 +1,10 @@
+#ifndef LLVM_TRANSFORMS_YK_NOMOREENTRYBLOCKCALLS_H
+#define LLVM_TRANSFORMS_YK_NOMOREENTRYBLOCKCALLS_H
+
+#include "llvm/Pass.h"
+
+namespace llvm {
+ModulePass *createYkNoMoreEntryblockCallsPass();
```

Comment:
If I had to nitpick, I'd say rename the pass to `NoCallsInEntryBlocks`.

Reply:
Or is `More` referencing that a malloc has to appear in an entry block?

Reply:
Yeah, our `malloc` is special since we insert it after this pass has run. Happy to rname to `NoCallsInEntryBlocks` though.

Reply:
OK. I wasn't sure on first glance if "No more" was just a quirky wording, or a significant wording.

I leave the decision up to you.

Reply:
[8e10f2c](https://github.com/ykjit/ykllvm/pull/105/commits/8e10f2c7da6f25b6e9f52764446228d146f41301)